testcert.py

```diff
@@ -837,412 +1001,411 @@ def testDeviceRegistrationInvalidUserAuthToken(self):
       raise
     try:
-      self.assertIn(Constants.PRINTER['MODEL'], device.cdd['model'])
+      self.assertIn(Constants.PRINTER['MODEL'], _device.cdd['model'])
     except AssertionError:
       notes = 'Printer model has unexpected value. Found %s' % (
-          device.cdd['model'])
+          _device.cdd['model'])
       self.LogTest(test_id, test_name, 'Failed', notes)
       raise
     else:
-      notes = 'Model: %s' % device.details['Model']
+      notes = 'Model: %s' % _device.details['model']
       self.LogTest(test_id, test_name, 'Passed', notes)
 
   def testPrinterManufacturer(self):
     """Verify printer provides a manufacturer string."""
     test_id = '68134ba3-5a05-4a77-82ca-b06ae6195cd8'
     test_name = 'testPrinterManufacturer'
     try:
-      self.assertIn('Manufacturer', device.details)
+      self.assertIn('manufacturer', _device.details)
     except AssertionError:
       notes = 'Manufacturer in not set in printer details.'
       self.LogTest(test_id, test_name, 'Failed', notes)
       raise
     try:
       self.assertIn(Constants.PRINTER['MANUFACTURER'],
-                    device.details['Manufacturer'])
+                    _device.details['manufacturer'])
     except AssertionError:
       notes = 'Manufacturer is not in printer details. Found %s' % (
-          device.details['Manufacturer'])
+          _device.details['Manufacturer'])
       self.LogTest(test_id, test_name, 'Failed', notes)
       raise
     try:
-      self.assertIn('manufacturer', device.cdd)
+      self.assertIn('manufacturer', _device.cdd)
     except AssertionError:
       notes = 'Manufacturer is not set in printer CDD.'
       self.LogTest(test_id, test_name, 'Failed', notes)
       raise
     try:
       self.assertIn(Constants.PRINTER['MANUFACTURER'],
-                    device.cdd['manufacturer'])
+                    _device.cdd['manufacturer'])
     except AssertionError:
       notes = 'Manufacturer not found in printer CDD. Found %s' % (
-          device.cdd['manufacturer'])
+          _device.cdd['manufacturer'])
       self.LogTest(test_id, test_name, 'Failed', notes)
       raise
     else:
-      notes = 'Manufacturer: %s' % device.details['Manufacturer']
+      notes = 'Manufacturer: %s' % _device.details['manufacturer']
       self.LogTest(test_id, test_name, 'Passed', notes)
 
   def testPrinterSerialNumber(self):
     """Verify printer provides a serial number."""
     test_id = '3996db1d-93ea-4f4c-b70c-dfd9355d5e5d'
     test_name = 'testPrinterSerialNumber'
     try:
-      self.assertIn('Serial Number', device.details)
+      self.assertIn('uuid', _device.details)
     except AssertionError:
       notes = 'Serial number not found in device details.'
       self.LogTest(test_id, test_name, 'Failed', notes)
       raise
     try:
-      self.assertGreaterEqual(len(device.details['Serial Number']), 1)
+      self.assertGreaterEqual(len(_device.details['uuid']), 1)
```

Comment:
That's good point. UUID is the field that is returned via the /printer GCP interface thus the code's been changed to use uuid as the key instead. I changed the test name to refer to UUID and added comments explaining its equivalence to SN.